Gopkg.toml

```diff
@@ -36,6 +36,10 @@ required = ["github.com/wadey/gocovmerge"]
   source = "github.com/simonferquel/containerd"
   revision = "42c3614d58767e96c1752977fed94d0b140a0173"
 
+[[override]]
+  name = "github.com/containerd/continuity"
+  revision = "bd77b46c8352f74eb12c85bdc01f4b90f69d66b4"
```

Comment:
we already depend on `simonferquel/containerd` (I would welcome a comment explaining why we need a fork) + a distinct commit. Highly confusing to me we mix stuff this way.